StarCellBio/settings.py

```diff
@@ -140,6 +146,7 @@
     'django.contrib.admindocs',
     'backend',
     'instructor',
+    'storages'
```

Comment:
I'm assuming this is using the `django-storages` package. Did you add that requirement to the `requirements.txt`?